xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/DXVAEnumeratorHD.cpp

```diff
@@ -440,3 +420,315 @@ bool CEnumeratorHD::IsSDRSupported()
 
   return true;
 }
+
+bool CEnumeratorHD::GetProcessorFormats(std::vector<DXGI_FORMAT>* inputFormats,
+                                        std::vector<DXGI_FORMAT>* outputFormats) const
+{
+  // Not initialized yet
+  if (!m_pEnumerator)
+    return false;
+
+  HRESULT hr;
+  UINT uiFlags;
```

Comment:
Initialize variables at declaration

Reply:
That one and the others are not used before initialization so it didn't seem necessary?